resources/factions/australia_2005.yaml

```diff
@@ -11,6 +11,9 @@ aircrafts:
   - F/A-18C Hornet (Lot 20)
   - SH-60B Seahawk
   - UH-1H Iroquois
+  - F/A-18E Super Hornet
+  - F/A-18F Super Hornet
+  - EA-18G Growler
```

Comment:
You're missing the banner, icon, and yaml file for the growler.

Reply:
changed

Reply:
banner and icon added